connector/ldap/ldap.go

```diff
@@ -68,7 +68,10 @@ type Config struct {
 
 	// Path to a trusted root certificate file.
 	RootCA string `json:"rootCA"`
-
+	// Path to a self-signed certificates cert file.
```

Comment:
Can we remove the "self-signed" there? I don't think it is necessary to have self-signed cert here.

Reply:
Sorry, I am not clear enough. The self-signed certificate is not dex. The ldap server uses a self-signed certificate (RootCA, server cert) and opens the certificate for the client. In this case, if dex Ability to authenticate client certificates through the ldap server, using client certificates generated by the Ldap server RootCA

Reply:
http://www.openldap.org/doc/admin24/tls.html
> 16.1.2. Client Certificates
The DN of a client certificate can be used directly as an authentication DN.
...

Reply:
without the client certificates, dex all requests to ldap server will be rejected.